### ISSUE TITLE
Open Science Community Barcelona - OSCBa

### ISSUE THREAD

**Issue by bonetcarne** [open-life-science/ols-1#16]
Project Lead: @bonetcarne
Mentor: @zebetus

Welcome to OLS-1! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

Before Week 1 (Jan 20): Meet your mentor!
- [X] Create an account on [GitHub](https://github.com)
- [x] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor
  
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.

- [X] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Jan 27): Cohort Call (Welcome to Open Life Science!)
- [x] Create an issue on the [OLS-1 GitHub repository](https://github.com/open-life-science/ols-1/issues/new) for your OLS work and share the link to your mentor.
- [x] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [x] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [x] Check the [Syllabus](https://openlifesci.org/ols-1) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 3): Meet your mentor!
- [x] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [ ] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

Before Week 4 (Feb 10): Cohort Call (Tooling and roadmapping for Open projects)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

Week 5 and more
- [x] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-1 Syllabus](https://openlifesci.org/ols-1) for more detailed weekly notes and assignments past week 4.

**Comment by cassgvp** [comment]
Hi @bonetcarne! So glad to see you in this program 😃. I think there is going to be a fair amount of overlap with the community building we'll both be doing, so let's stay close and maybe develop some ideas together? Check me out on #15 👀😝

**Comment by bonetcarne** [comment]
> Hi @bonetcarne! So glad to see you in this program 😃. I think there is going to be a fair amount of overlap with the community building we'll both be doing, so let's stay close and maybe develop some ideas together? Check me out on #15 👀😝

So glad to see you here as well! Definitely we should stay close and work together. :)

**Comment by bonetcarne** [comment]
I am working with @Zebetus to create and hopefully establish an Open Science Community in Barcelona to promote knowledge exchange and will discuss on how we could improve our research in terms of open science. Long term aim is  to use this community to create workshops and symposia about open science and to agree on some common needs.

**Comment by Karvovskaya** [comment]
Hi @bonetcarne! I am very excited about your project. I am doing something similar. There is an Open Science Community in Amsterdam, and I want to strengthen it with a network of Open Science and data ambassadors (data champions). Let's work together and help each other :-)

**Comment by bonetcarne** [comment]
> Hi @bonetcarne! I am very excited about your project. I am doing something similar. There is an Open Science Community in Amsterdam, and I want to strengthen it with a network of Open Science and data ambassadors (data champions). Let's work together and help each other :-)

Great! are you part of the Netherlands OS communitry, like https://openscience-utrecht.com ?
Even if I am not based on the Netherlands, I was panning to contact them, they seem quite organised and provide good tips on how to start one community (https://openscience-utrecht.com/community-blueprint/) :)

**Comment by SamGuay** [comment]
Hi @bonetcarne ! 

I think we are trying to accomplish the same thing (but on the other side of the Atlantic)! One great resource that has guided me is from the [Melbourne Open Research Network](https://www.melbourneopenresearch.org/). This [thread](https://twitter.com/lingtax/status/1063237416985579520) and their [OSF repo](https://osf.io/be7yt/wiki/home/) in particular.

[OSCU](https://openscience-utrecht.com) leaders are responsive and really helpful. I contacted them last year and they got back soon after. 

Will your community be geared toward particular fields or you want it to be as broad as possible? Both are okay, just different challenges. I'm guessing you are a neuroscientist if you know about the Open Science Beverage (or beer if you prefer) that happens every Wednesdays in Montreal.

**Comment by bonetcarne** [comment]
Hi @SamGuay !
Great, indeed looks we are trying similar things, great! The idea is to be as broad as possible but I guess this is going to be a life thing, so it will evolve and maybe end up focusing on one specific. But the idea is to include researchers from different universities and disciplines.
I did neuroscience but mostly know about them through diffusion MRI and some side sessions at ISMRM (secret sessions), where there was a lot of discussions about open and inclusive science. :)

What about your project? For what I read is also broad, isn't it?

**Comment by SamGuay** [comment]
Yes, we aim to reach many fields because the neuroimaging community may be at the forefront on some levels in OS, but we can definitely learn from other fields and vice versa. This will be challenging, but that's why we're gonna have to organize different types of activities to target several audiences. We are thinking about setting up a [ReproducibiliTea](https://reproducibilitea.org/) Journal club. 

I'm doing dMRI as well, with the SCIL/Tractoflow people at USherbrooke (Maxime Descoteaux). 

Will I see you this summer at OHBM?

**Comment by ChiaraBertipaglia** [comment]
Hi @bonetcarne and @Karvovskaya, I was just exploring the data champions initiative of TUDelft https://www.tudelft.nl/en/library/current-topics/research-data-management/r/support/data-champions/. Since I am at such early stages compared to these awesome examples, I also am interested in knowing how they got it started at all. How didi they bubble up interest and engagement?

**Comment by Karvovskaya** [comment]
Hi @ChiaraBertipaglia, I can put you in touch with TU Delft people if you are interested. Here is a nice interview where two of the earliest data champions share their perspective: https://openworking.wordpress.com/2019/08/01/carve-your-niche-with-the-carpentries/ By the way, have you already looked at this book https://www.openbookpublishers.com/product/1080? It has a chapter on data champions.

**Comment by Karvovskaya** [comment]
> are you part of the Netherlands OS community, like https://openscience-utrecht.com ?

@bonetcarne, yes, I am a member of OSC Utrecht and I am planning to get more involved in the OSC Amsterdam. From what I know, the organisation of open science communities and their positioning within universities differ quite a lot. 

> I was panning to contact them, they seem quite organised and provide good tips on how to start one community (https://openscience-utrecht.com/community-blueprint/) :)

I definitely recommend to contact them. For Utrecht, my suggestion for contact would be Loek Brinkman: https://loekbrinkman.com/ For Rotterdam, Antonio Schettino

**Comment by ChiaraBertipaglia** [comment]
Thank you @Karvovskaya that book is very useful! Yes, I'd love to be put in contact with someone from TuDelft. Thanks!

**Comment by malvikasharan** [comment]
Love this thread and so many collaborations + idea exchange! Great going @bonetcarne @cassgvp @ChiaraBertipaglia @Karvovskaya @SamGuay. There are a few more projects in the cohort for e.g. Open Connect Kenya by @Lilian9, and the projects by @MorphoFun and @Fnyasimi that share some similar vision.

**Comment by cassgvp** [comment]
A project I've been considering is profiling or collecting case studies about these open communities. Focusing on things like how/why they originated, how they are funded or sustained, what types of things they organise, who their stakeholders are and how they gained traction. All the same questions I think we've all had! How would people feel about generating a list of questions together and then doing structured interviews with existing initiatives? Could publish them in a nice blog somewhere? We could even profile ourselves at various stages?! 😁

**Comment by malvikasharan** [comment]
Hi Project Leads,  

This is your project report file: 
https://hackmd.io/W6xj5P_3SMSGNi_M0D8fHA
Please start working on it for the final presentation. We will send more info in the weekly email.  

Best,
 OLS team
